skribenten-backend/src/test/kotlin/no/nav/pensjon/brev/skribenten/services/BrevredigeringServiceTest.kt

```diff
@@ -489,7 +489,7 @@ class BrevredigeringServiceTest {
         val brev = opprettBrev().resultOrNull()!!
 
         awaitAll(*(0..<10).map {
-            async(Dispatchers.IO) {
+            async {
```

Comment:
Spørsmålet er om vi faktisk får testet det vi ønsker? Målet er å fyre av transaksjoner som treffer samme rad parallellt.

Reply:
Dispatchers.IO har et større thread-pool enn Dispatchers.Default, slik jeg tolker det. Så spørsmålet er om testen feiler fordi det vi ønsker å teste faktisk skjer, eller om et er en annen grunn.

Reply:
Så det som potensielt avsløres her er at vi har to transaksjoner som treffer ish samtidig. Slik at en av transaksjonene opplever at findSingleByAndUpdate ikke fant noe, men innen den treffer Document.new så finnes en rad med samme ID.

```
Document.findSingleByAndUpdate(DocumentTable.brevredigering eq brevredigering.info.id, update)?.pdf?.bytes
        ?: Document.new(update).pdf.bytes
```